src/flake8_comprehensions/__init__.py

```diff
@@ -199,6 +200,21 @@ def run(self) -> Generator[tuple[int, int, str, type[Any]], None, None]:
                         type(self),
                     )
 
+                elif (
+                    node.func.id == "sorted"
```

Comment:
From the original issue:
> the (a for a in b) generator construct doesn't seem particularly useful and just adds an unnecessary context switch of the generator.

I think we can check every generator, not only those inside sorted call.
This mean that the check would be only the `isinstance` of node as `ast.GeneratorExp` and `isinstance` of `node.elt` as `ast.Name`. What do you think?

Reply:
You should be right.
We should verify if the Generator do something, in all cases, not only for sorted call.
I need to put this on my desktop : KISS 👍